src/containers/Articles/index.jsx

```diff
@@ -33,21 +33,22 @@ const Articles = ({
   articlesLoading,
   articlesError
 }) => {
-  if (loading) {
-    return 'Loading';
+  if (dataError) {
+    return <div> Couldin't Fetch articles data </div>;
   }
 
-  if (dataError) {
-    return 'error';
+  if (loading) {
+    return 'Loading......';
   }
-  if (data && articlesData) {
+
+  if (data.length) {
```

Comment:
try not to assume that people know that a length of zero would coerce to false here, go with `data.length > 0` as it's more explicit. always try to be more explicit when writing your code